sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -27,12 +27,68 @@
 )
 import pytest
 import datetime
+import os
+import logging
 
-class AppConfigurationClientTest(AzureAppConfigurationClientTestBase):
+class AppConfigurationClientTest(AzureMgmtTestCase):
     def __init__(self, method_name):
-        super(AppConfigurationClientTest, self).__init__(
-            method_name, AzureAppConfigurationClient
+        super(AppConfigurationClientTest, self).__init__(method_name)
+        self.vcr.match_on = ["path", "method", "query"]
+        if self.is_playback():
+            connection_str = "Endpoint=https://fake_app_config.azconfig-test.io;Id=0-l4-s0:h5htBaY5Z1LwFz50bIQv;Secret=bgyvBgwsQIw0s8myrqJJI3nLrj81M/kzSgSuP4BBoVg="
+        else:
+            connection_str = os.getenv('APP_CONFIG_CONNECTION')
```

Comment:
You can do something like this:
```python
self.connection_str = self.set_value_to_scrub("APP_CONFIG_CONNECTION", APP_CONFIG_CONNECTION)
```
That will read the env variable AZURE_APP_CONFIG_CONNECTION, and if not present will read APP_CONFIG_CONNECTION in the settings file, and raise if not present + it will scrub it automatically if the value is seen in the recordings.

Example:
https://github.com/Azure/azure-sdk-for-python/blob/8064bc5df9a3c086fc17572cb7a96d4b0fa5d1b4/sdk/keyvault/azure-keyvault-certificates/tests/certificates_preparer.py#L99

Reply:
Also, can't you create the AppConfig using ARM? Like Storage and KeyVault are doing in their test? This ensure the AppConfig is deleted correctly.

Reply:
Will be done in https://github.com/Azure/azure-sdk-for-python/issues/9062